package.json

```
@@ -16,6 +16,10 @@
   "repository": "https://github.com/xjamundx/eslint-plugin-promise",
   "homepage": "https://github.com/xjamundx/eslint-plugin-promise",
   "bugs": "https://github.com/xjamundx/eslint-plugin-promise/issues",
+  "main": "index.js",
+  "files": [
+    "rules"
```

```suggestion
    "rules/**/*.js",
    "index.js"
```

I would go with this

---

that would miss [rules/lib/is-member-call-with-object-name](https://github.com/eslint-community/eslint-plugin-promise/blob/main/rules/lib/is-member-call-with-object-name) which perhaps does want to end with .js?

---

Yes, sorry, that does want to end with `.js`. Addressed in the TypeScript PR, but I can supply a separate PR to address.